src/test/java/com/thomashan/coup/PlayerBuilder.java

```diff
@@ -3,43 +3,73 @@
 import static com.thomashan.coup.Card.AMBASSADOR;
 
 public class PlayerBuilder {
-    public static Player build(boolean active) {
-        Player player = build();
-        if (active) {
-            return player;
-        }
+    private int coins = 2;
+    private boolean active = true;
+    private Card card1 = AMBASSADOR;
+    private Card card2 = AMBASSADOR;
+
+    public static PlayerBuilder newBuilder() {
+        return new PlayerBuilder();
+    }
 
-        return player.revealCard1().revealCard2();
+    public PlayerBuilder coins(int coins) {
+        this.coins = coins;
+        return this;
     }
 
-    public static Player build() {
-        return build(2);
+    public PlayerBuilder active(boolean active) {
+        this.active = active;
+        return this;
     }
 
-    public static Player build(int coins) {
-        Player player = createPlayerWithNumberOfCoins(coins);
+    public PlayerBuilder card1(Card card1) {
+        this.card1 = card1;
+        return this;
+    }
+
+    public PlayerBuilder card2(Card card2) {
+        this.card2 = card2;
+        return this;
+    }
 
-        assert player.getCoins() == coins;
+    public Player build() {
+        Player player = Player.of(card1, card2);
+        player = coins(player, coins);
+        player = active(player, active);
 
         return player;
     }
 
-    private static Player createPlayerWithNumberOfCoins(int coins) {
-        Player player = Player.of(AMBASSADOR, AMBASSADOR);
+    private Player active(Player playerToRevealCards, boolean active) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'UR'-anomaly for variable 'playerCard' (lines '43'-'51').](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)